fblock.go

```diff
@@ -77,7 +77,7 @@ type FBlockHeader struct {
 func (fb FBlock) IsPopulated() bool {
 	return len(fb.Transactions) > 0 && // FBlocks always contain at least a coinbase
 		fb.Header.BodyMR != nil &&
-		fb.Header.PrevKeyMR != nil && // TODO: Why not check the computed keymr?
```

Comment:
Is populated is primarily used for two purposes: ensuring that MarshalBinary can proceed and allowing Get to exit early so as to avoid making redundant network calls. So what is included here should be chosen with that in mind. KeyMR is not needed to marshal.